src/OpenTelemetry/BatchExportProcessorOptions.cs

```diff
@@ -26,7 +26,7 @@ public class BatchExportProcessorOptions<T>
     public int ExporterTimeoutMilliseconds { get; set; } = BatchExportProcessor<T>.DefaultExporterTimeoutMilliseconds;
 
     /// <summary>
-    /// Gets or sets the maximum batch size of every export. It must be smaller or equal to MaxQueueLength. The default value is 512.
+    /// Gets or sets the maximum batch size of every export. It must be smaller or equal to MaxQueueSize. The default value is 512.
```

Comment:
We can link this. In case of any further changes, it should be detected by the build tools.

```suggestion
    /// Gets or sets the maximum batch size of every export. It must be smaller or equal to <see cref="MaxQueueSize"/>. The default value is 512.
```